docs/install/install-kn.md

```diff
@@ -13,13 +13,33 @@ To install the `kn` Client, you must download the executable binary for your sys
 You must place the executable binary in your system path, and make sure that it is executable.
 
 ## Install `kn` using Go
-1. Check out the [Client repository](https://github.com/knative/client).
-1. Run the command:
+1. Build `kn` requires Go v1.13 or newer. Please install [go](https://golang.org/doc/install) environment firstly.
+1. Download the [Client repository](https://github.com/knative/client).
+  ```bash
+  cd $GOPATH/src/knative-dev
+  git clone https://github.com/knative/client.git
+  cd client/
+  ```
+1. Build an executable binary:
   ```bash
   go install ./cmd/kn
   ```
+1. Move `kn` into your system path, and verify whether kn is working properly. For example:
+  ```bash
+  cp $GOPATH/bin/kn /bin/
+  kn version
+  ```
 
 ## `kn` container images
 The `kn` container images are available here:
 - [Nightly container image](https://gcr.io/knative-nightly/knative.dev/client/cmd/kn)
 - [Latest release](https://gcr.io/knative-releases/knative.dev/client/cmd/kn)
+
+You can run `kn` from container. For example:
+  ```bash
+  docker pull https://gcr.io/knative-releases/knative.dev/client/cmd/kn
+  docker run --rm --name <your-container-name> -v "$HOME/.kube/config:<your-kubeconfig-path>" gcr.io/knative-releases/knative.dev/client/cmd/kn:latest service list
```

Comment:
* No need to select a name (just leave that out)
* Instead of `<your-kubeconfig-path>` we need to provide the concrete directory.

Still have to figure that out, but @navidshaikh how is this image called in the tekton integration ?

In general it doesnt make sense to add placeholder in an installation intstructions if it is not explained what they should be (and of course shouldn't be added if not needed)

Reply:
We don't need to touch anything for kubeconfig while working with tekton. 

IMO, we should rmeove this command example and add a sub-section here for for e.g.: 'Using kn with Tekton' which links to the kn catalog task at https://github.com/tektoncd/catalog/tree/master/kn

Reply:
May be we can both introduce `Running kn in docker` and `Running kn with tekton task`. For some user, they are not familiar with tekton.